bucket/apimtemplate.json

```
@@ -0,0 +1,20 @@
+{
+    "homepage": "https://github.com/Azure/azure-api-management-devops-resource-kit",
+    "description": "Azure API Management DevOps Resource Kit",
+    "license": "MIT",
+    "version": "0.1-alpha",
+    "url": "https://github.com/Azure/azure-api-management-devops-resource-kit/releases/download/v0.1-alpha/azure-api-management-devops-example-win-x64-v0.1-alpha.zip",
+    "hash": "8b980e59eec011c5eed042e8dc576b76e22b871f8dccefec6e4181dfcc281889",
+    "depends": "azure-cli",
+    "bin": [
+        "apimtemplate.exe"
+    ],
+    "checkver": "github",
```

```suggestion
    "checkver": {
        "url": "https://github.com/Azure/azure-api-management-devops-resource-kit/releases",
        "regex": "tag/(?:v)?([\\w.-]+)"
    },
```

It is prerelease, `github` mode is not suitable.

---

done